pkg/filters/builder/requestadaptor/requestadaptor.go

```diff
@@ -248,9 +254,24 @@ func (ra *RequestAdaptor) Handle(ctx *context.Context) string {
 	req := ctx.GetInputRequest().(*httpprot.Request)
 	method, path := req.Method(), req.Path()
 
-	if ra.spec.Method != "" && ra.spec.Method != method {
-		ctx.AddTag(stringtool.Cat("requestAdaptor: method ", method, " adapted to ", ra.spec.Method))
-		req.SetMethod(ra.spec.Method)
+	tempSpec := &Spec{}
```

Comment:
`tempSepc` is not a good name, does `temp` mean `template` or `temporary`?

Reply:
will rename it to templateSpec